manifests/v/ViviCorporation/Vivi/3.3.6/ViviCorporation.Vivi.installer.yaml

```
@@ -0,0 +1,32 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: ViviCorporation.Vivi
+PackageVersion: 3.3.6
+InstallerLocale: en-US
+MinimumOSVersion: 10.0.0.0
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+InstallerSwitches:
+  Silent: /qn
+  SilentWithProgress: /qb
+UpgradeBehavior: install
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: msi

```

```suggestion
  InstallerType: wix
```
InstallerType differences in new manifests can cause issues in upgrade scenario for the package. Although I'm almost sure setting this as `msi` here wouldn't be all that harmful; its better to keep it as its [previous value](https://github.com/microsoft/winget-pkgs/blob/200efa2a5e56552acffd8c3cfbba29a982667578/manifests/v/ViviCorporation/Vivi/3.2.0/ViviCorporation.Vivi.installer.yaml#L8).